credentials/bc-petroleum-and-natural-gas-title/examples/v0-embedded-proof.jsonld

```diff
@@ -0,0 +1,141 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2",
+    "https://untp.traceability.site/core/dcc/v0.jsonld",
+    "https://untp.traceability.site/bcgov/tt/v0.jsonld",
+    "https://w3id.org/security/data-integrity/v2",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
+  ],
+  "id": "https://bcgov.github.io/bc-vcpedia/credentials/bc-petroleum-and-natural-gas-title/examples/v0.jsonld",
+  "type": [
+    "VerifiableCredential",
+    "ExampleCredential",
+    "DigitalConformityCredential",
+    "BCTenureTitle"
+  ],
+  "issuer": {
+    "id": "did:web:bcgov.github.io:bc-vcpedia:applications:tenure-branch",
+    "url": "https://www2.gov.bc.ca/gov/content/industry/natural-gas-oil/petroleum-natural-gas-tenure",
+    "name": "Tenure and Resource Stewardship Branch",
+    "description": "The Tenure and Resource Stewardship Branch (TRSB)"
+  },
+  "credentialSubject": [
+    {
+      "type": [
+        "ConformityAttestation"
+      ],
+      "issuedTo": {
+        "type": [
+          "LegalEntity",
+          "TitleHolder",
+          "Lessee"
+        ],
+        "id": "https://orgbook.gov.bc.ca/entity/A0131571",
+        "legalName": "PACIFIC CANBRIAM ENERGY LIMITED",
+        "interest": 100,
+        "identifier": "A0131571"
+      },
+      "scope": {
+        "type": [
+          "ConformityAssessmentScheme"
```

Comment:
Discussion point - needs further clarification from UNTP regarding linkage of assessment scheme to regulation. Will assume for the purpose of demo that this artifact relates to the governance of the credential (application layer).